Automattic/wp-calypso
# Stats: Release the new annual highlights section

Note: Consider waiting for [these strings to be fully translated](https://translate.wordpress.com/deliverables/overview/7610530/) before release.

#### Proposed Changes

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/4044428/201448004-beba7e08-da6a-453d-a9f0-22ffcbd5bfbe.png">

* Adds a new annual highlights section at the top of the Insights page.
* Hides the existing "Annual site stats" module at the bottom left of the Insights page.

#### Testing Instructions



* Spin up this change in a live branch.
* Navigate to Stats -> Insights.
* Ensure you see the new annual highlights section.

#### Pre-merge Checklist






Related to #69227
